x-pack/plugins/apm/server/routes/settings/agent_configuration.ts

```diff
@@ -143,11 +143,11 @@ export const agentConfigurationSearchRoute = createRoute(core => ({
   params: {
     body: t.intersection([
       t.type({ service: serviceRt }),
-      t.partial({ etag: t.string })
+      t.partial({ etag: t.string, appliedByAgent: t.boolean })
```

Comment:
The server serializes json with underscores by default, which results in `applied_by_agent`; 
I created https://github.com/elastic/apm-server/pull/3677, but something seems to be off as `applied_by_agent` is always `false` in Kibana's response and the UI does not show the config as applied in the UI (also when changing APM server code to serialize the flag as `appliedByAgent`).

Reply:
Changed to `applied_by_agent` 42c85ab